.github/workflows/build.yml

```diff
@@ -86,19 +86,21 @@ jobs:
 
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@5ecb98a3c6b747ed38dc09f787459979aebb39be # v4.3.1
+      if: always() && github.event.repository.fork == false
```

Comment:
Please revert all the changes except this one - if someone wants to run our builds in their fork, then they should be able to access the artifacts produced by them.